python/rascaline/rascaline/utils/splines.py

```diff
@@ -0,0 +1,403 @@
+"""
+Splined radial integrals
+========================
+
+Classes for generating spline points which can be used as input for the for the radial
+integral hyper paramater option. For an complete example for the LE how to use the
+classes see :ref:`example-splines`.
+
+.. autoclass:: rascaline.utils.RadialIntegralSplinerBase
+    :members:
+    :show-inheritance:
+
+.. autoclass:: rascaline.utils.RadialIntegralFromFunction
```

Comment:
we should pick a place for these. I see you like to have them in the module docstring, and I prefer them in the docs sources. Let's chat about it in the meeting later today?